ghcjs-base.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       3.0
 name:                ghcjs-base
-version:             0.2.1.0
+version:             1.0.0.0
```

Comment:
@hamishmack suggested this on slack a while ago as a way to break version bounds with projects using GHCJS. IMO, there's also the possibility of releasing with a new name, but I don't think it's necessary.

Reply:
I think the problem with a new name is that they should be mutually exclusive.  If we keep the same name there is no way anyone will wind up with both in their build plan.

I wonder if we should use 0.8 or something to avoid confusion.  It seems unlikely we will want to release any more non `arch(javascript)` versions.